src/IW/Db/Issue.hs

```diff
@@ -4,48 +4,49 @@
 
 module IW.Db.Issue
        ( getIssues
-       , getIssuesByLabel
+       , getIssuesByLabels
        , upsertIssues
        ) where
 
-import IW.Core.Issue (Issue (..))
+import IW.Core.Issue (Issue (..), Label (..))
+import IW.Core.SqlArray (SqlArray (..))
 import IW.Core.WithId (WithId)
 import IW.Db.Functions (WithDb, executeMany, query, queryRaw)
 
 
--- | Returns all issues in the database
+-- | Returns all issues in the database.
 getIssues :: (WithDb env m) => m [WithId Issue]
 getIssues = queryRaw [sql|
     SELECT id, repo_owner, repo_name, number, title, body, labels
     FROM issues
     LIMIT 100
 |]
 
--- | Returns a list of issues filtered by label name
-getIssuesByLabel :: (WithDb env m) => Text -> m [WithId Issue]
-getIssuesByLabel = query [sql|
+-- | Returns all issues with at least one label in the given @SqlArray@.
+getIssuesByLabels :: (WithDb env m) => SqlArray Label -> m [WithId Issue]
+getIssuesByLabels = query [sql|
     SELECT id, repo_owner, repo_name, number, title, body, labels
-    FROM issues 
-    WHERE ? = ANY (labels)
+    FROM issues
+    WHERE ? && labels
```

Comment:
Can you write an explanation on how this works or provide a link to the documentation? :slightly_smiling_face: I'm just curious

Reply:
@chshersh I found this right here: 
https://stackoverflow.com/questions/21742929/postgresql-check-if-array-contains-any-element-from-left-hand-array
If ANY element in one array is in the other it returns true.